chain/xrp/tx/tx_test.go

```diff
@@ -0,0 +1,62 @@
+package tx_test
+
+import (
+	"encoding/hex"
+	"testing"
+
+	xc "github.com/cordialsys/crosschain"
+	"github.com/cordialsys/crosschain/chain/xrp/tx"
+	"github.com/test-go/testify/require"
+)
+
+func TestTxHash(t *testing.T) {
+
+	tx1 := tx.Tx{}
+	hash := tx1.Hash()
+	require.Equal(t, "", string(hash))
+}
+
+func TestTxSighashes(t *testing.T) {
```

Comment:
I don't think this is a very good test.  it's basically testing it doesn't return an error.

I think it'd be much more useful to have test vectors with more realistic transactions and assert that their sighashes are expected values.